FoundationsMastery/BreakfastDrink.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace FoundationsMastery
+{
+    public class BreakfastDrink
+    {
+        public string Caffeine { get; set; }
+        public string Juice { get; set; }
+        
+        public virtual void defaultCaffeine()
+        {
+            string coffee = "I'll have coffee.";
+        }
+
+        public virtual void defaultJuice()
+        {
+            string juice = "Orange Juice.";
```

Comment:
`tea`, `grapefruit`, `coffee`, & `juice` are all local variables in these methods that don't get returned - perhaps you should set the public properties `Caffeine` and `Juice`, and add a public string `Tea` to the derived class?